internal/gensupport/resumable.go

```diff
@@ -188,6 +192,14 @@ func (rx *ResumableUpload) Upload(ctx context.Context) (resp *http.Response, err
 	// Configure retryable error criteria.
 	errorFunc := rx.Retry.errorFunc()
 
+	// Configure per-chunk transfer timeout.
+	var transferTimeout time.Duration
+	if rx.ChunkTransferTimeout != 0 {
+		transferTimeout = rx.ChunkTransferTimeout
+	} else {
+		transferTimeout = defaultTransferTimeout
```

Comment:
I think since the default is zero, we don't need this if/else?

Reply:
I kept it this way to make it easier to update the default value from 0 to something else in the future.  Anyways updated the changes as you said.